controller/argument_value_resolver.rst

```diff
@@ -51,12 +55,12 @@ controller::
 
     namespace AppBundle\Controller;
 
-    use AppBundle\Entity\User;
     use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\Security\Core\User\UserInterface;
 
     class UserController
     {
-        public function indexAction(User $user)
+        public function indexAction(UserInterface $user)
```

Comment:
I disagree, the implementation you suggest is already done in the security bundle as of 3.2 (see https://github.com/symfony/symfony/blob/3.2/src/Symfony/Bundle/SecurityBundle/SecurityUserValueResolver.php), so it would not be custom anymore.